### PR TITLE
Document which tracking parameters are required in README.

### PR DESCRIPTION
**Pull request by sblaurock**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,11 +176,11 @@ constructorio.tracker.trackSearchResultClick('dogs', {
...
-| Parameter | Type |
-| --- | --- |
-| `name` | string |
-| `customer_id` | string |
-| `result_id` | string |
+| Parameter | Type | Required |
+| --- | --- | :---: |
+| `name` | string | ✔ |
+| `customer_id` | string | ✔ |
+| `result_id` | string | ✔ |
```
Comment (rubikzube) [inline README.md:183]: `result_id` is not required,  more of a very nice to have

File: README.md [hunk 2]
```diff
@@ -189,13 +189,13 @@ constructorio.tracker.trackConversion('dogs', {
...
-| `name` | string |
-| `customer_id` | string |
-| `result_id` | string |
-| `revenue` | string |
-| `section` | string |
+| Parameter | Type | Required |
+| --- | --- | :---: |
+| `name` | string | ✔ |
+| `customer_id` | string | ✔ |
+| `result_id` | string | ✔ |
```
Comment (rubikzube) [inline README.md:196]: `result_id` is not required,  more of a very nice to have

File: README.md [hunk 3]
```diff
@@ -189,13 +189,13 @@ constructorio.tracker.trackConversion('dogs', {
...
+| `revenue` | string | ✔ |
+| `section` | string | ✔ |
```
Comment (rubikzube) [inline README.md:198]: `section` is not required,  should default to `Products`

File: README.md [hunk 4]
```diff
@@ -204,11 +204,11 @@ constructorio.tracker.trackPurchase({
...
-| Parameter | Type |
-| --- | --- |
-| `customer_ids` | string |
-| `revenue` | string |
-| `section` | string |
+| Parameter | Type | Required |
+| --- | --- | :---: |
+| `customer_ids` | string | ✔ |
+| `revenue` | string | ✔ |
+| `section` | string | ✔ |
```
Comment (rubikzube) [inline README.md:211]: `section` is not required, should default to `Products`

File: README.md (hunk 1 above)
Comment (sblaurock) [reply README.md]: Great catches here - updated the JSDoc as well.

**Pull request merged**